### PR TITLE
[components-webview] Use common sharing page. Contributes to JB#41068

### PR DESCRIPTION
**Pull request by pvuorela**
@rainemak

### CODE REVIEW

File: import/popups/ShareTextPage.qml [hunk 1]
```diff
@@ -13,40 +13,20 @@ import QtQuick 2.0
...
-        content: {
-            "name": text,
-            "data": text,
-            "type": type,
-            "icon": icon,
-            // also some non-standard fields for Twitter/Facebook status sharing:
-            "status" : text,
-            "linkTitle" : text
-        }
-        filter: type
```
Comment (rainemak) [inline import/popups/ShareTextPage.qml]: What happens/happened to this?

Comment (rainemak) [reply import/popups/ShareTextPage.qml]: Answering to myself mimeType is a sharemethodlist.filter alias.

File: import/popups/ShareTextPage.qml [hunk 2]
```diff
@@ -13,40 +13,20 @@ import QtQuick 2.0
...
-        header: PageHeader {
-            //: List header for sharing method list (generic)
-            //% "Share"
-            title: qsTrId("sailfish_components_webview_popups-he-share")
-        }
-        content: {
-            "name": text,
-            "data": text,
-            "type": type,
-            "icon": icon,
```
Comment (rainemak) [inline import/popups/ShareTextPage.qml]: Don't recall but is this one of those deprecated fields?

Comment (pvuorela) [reply import/popups/ShareTextPage.qml]: Haven't spotted any usage and no one else seems to have idea why this has been there :)

**Pull request merged**